draft-ietf-tls-external-psk-guidance.md

```diff
@@ -176,17 +190,18 @@ overall system is inherently rather brittle. There are a number of
 obvious weaknesses here:
 
 1. Any group member can impersonate any other group member.
-1. If PSK with DH is used, then compromise of a group member that actively
-completes connections with other group members can read (and modify) traffic.
-1. If PSK without DH is used, then compromise of any group member allows the
-attacker to passively read (and modify) all traffic.
-1. If a group member is compromised, then the attacker can perform all of the above attacks.
+1. If PSK is combined with DH, then compromise of a group member that knows
+the resulting DH shared secret will enable the attacker to read (and modify) traffic.
+1. If PSK is not combined with DH, then compromise of any group member allows the
+attacker to passively read (and actively modify) all traffic.
 
 Additionally, a malicious non-member can reroute handshakes between honest group members
-to connect them in unintended ways, as described below. Note that this class of attack is
-not possible if each member uses the SNI extension {{!RFC6066}} and terminates the
-connection on mismatch between the presented SNI value and the receiving member's
-known identity. See {{Selfie}} for details.
+to connect them in unintended ways, as described below. Note that this class of attack
```

Comment:
I'd revert this text too. @kaduk's comment on selfie doesn't seem to be in the scope of that attack, so IMO the previous text more accurately describes things.

Reply:
I don't think that even strict use of SNI renders the class of attack "not possible", full stop.
But your point that this is tangential and we may not need to go into quite so much detail seems reasonable.

Reply:
I do not see how reverting addresses the original comment.